sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/ViewRegistry.java

```diff
@@ -29,19 +29,22 @@
       View.builder()
           .setAggregatorFactory(AggregatorFactory.sum(AggregationTemporality.CUMULATIVE))
           .build();
-  static final View SUMMARY =
-      View.builder().setAggregatorFactory(AggregatorFactory.minMaxSumCount()).build();
+  static final View DEFAULT_HISTOGRAM =
+      View.builder()
+          .setAggregatorFactory(
+              AggregatorFactory.histogram(
+                  Arrays.asList(
+                      5d, 10d, 25d, 50d, 75d, 100d, 250d, 500d, 750d, 1_000d, 2_500d, 5_000d,
+                      7_500d, 10_000d),
```

Comment:
It feels like this should be a constant somewhere.

Reply:
Move it locally for now.  Will shift as Aggregator re-work occurs.